internal/store/builder.go

```diff
@@ -91,6 +94,11 @@ func (b *Builder) WithKubeClient(c clientset.Interface) {
 	b.kubeClient = c
 }
 
+// WithVPAClient sets the vpaClient property of a Builder so that the verticalpodautoscaler collector can query VPA objects
```

Comment:
Comments need to end with full stops.
```suggestion
// WithVPAClient sets the vpaClient property of a Builder so that the verticalpodautoscaler collector can query VPA objects.
```